hare/eligibility/beacon.go

```diff
@@ -21,23 +21,24 @@ type addGet interface {
 	Get(key interface{}) (value interface{}, ok bool)
 }
 
-type beacon struct {
+// Beacon provides the value that is under consensus as defined by the hare.
+type Beacon struct {
```

Comment:
you exported it because it is used outside the pkg ?

Reply:
`NewBeacon()` is used outside the package (in `node.go`) and it returns the beacon.

Reply:
What Noam said (an exported method that returns an unexported type).